scripts/run_backend_tests.py

```diff
@@ -493,14 +486,22 @@ def main(args=None):
             '%s errors, %s failures' % (total_errors, total_failures))
 
     if parsed_args.generate_coverage_report:
-        subprocess.check_call(['python', COVERAGE_PATH, 'combine'])
-        subprocess.check_call([
-            'python', COVERAGE_PATH, 'report',
-            '--omit="%s*","third_party/*","/usr/share/*"'
-            % common.OPPIA_TOOLS_DIR, '--show-missing'])
-
-        python_utils.PRINT('Generating xml coverage report...')
-        subprocess.check_call(['python', COVERAGE_PATH, 'xml'])
+        subprocess.check_call(['coverage', 'combine'])
+        process = subprocess.Popen(
+            ['coverage', 'report',
```

Comment:
Again, should probably use a local version of coverage here.

Reply:
Reverted it back to the local version.